File: Assignment No_27/Question_5.c
/*5.Write a program which accept string from user and display it in reverse order.

Input : "MarvellouS"
Output : "SuollevraM"

Input : "Ganesh Gawali"
Output: "ilawaG hsenaG"

*/
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#include<string.h>

void Reverse(char *str)
{
    int i=0;

    for(i=strlen(str); i>=0; i--)
    {
        printf("%c",str[i]);
    }
    printf("\n");
    
}

int main()
{
   char Arr[20];
   int iRet=0;

   printf("Enter the string:\n");
   scanf("%[^'\n']s",Arr);

   Reverse(Arr);

   return 0;
}

File: Assignment No_36/Question_5.java
/*5. Write a java program which accept string from user and display it in reverse order.

Input : “MarvellouS”
Output : “SuollevraM”

Input : "Ganesh Gawali"
Output: "ilawaG hsenaG"

*/
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Ass36_5
{
    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter The String : ");
        String name = sobj.nextLine();

        int i = 0;
        char ch;

        System.out.print("Reversed String: ");

        for(i = name.length()-1; i >=0; i--)
        {

           System.out.print(name.charAt(i));   
           
        }        
 
    }
}



